tracks/ruby/exercises/acronym/readme.md

```diff
@@ -0,0 +1,44 @@
+# Acronym
+
+_Acronym_ is one of the first side exercises, disclosed by _TwoFer_
+
+## Reasonable Solutions
+
+```ruby
+module Acronym
+  def self.abbreviate(fullname)
+    fullname.scan(/\b[a-zA-Z]/).join.upcase
+  end
+end
+```
+
+## Common suggestions
+* Most first submissions split the phrase into words with `String#split`: 
+  ```ruby
+  phrase.split(/[ -]/)
+  ```
+  , followed by operations to find the first letters.
+  Variation: first `gsub` or `tr` the non-space delimiters, then `split`
+  
+In all these cases: point to String#scan.   
+* The first use of `scan` mostly is a scan for full words. That's a perfect moment to show the power of `scan`: 
+when it returns the first letters only, it will save a few operations.  
+* As this exercise is not about regex's, it's recommended to 
+give the regex away for free, especially if they already found `scan(/\w+/)`
+
+## Talking points
+* If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
+* `scan` is faster than `split`
+* It can be hard to catch all the possible delimiters. 
+* `upcasing` the whole string is 10% slower than `upcasing` the acronym only. 
```

Comment:
citation needed?

Reply:
"Jeremy told me"..... 🤔 
I didn't research this. Yet.

Reply:
Saying something more vague like "upcasing the whole string will be slower than only upcasing the acronym" might be better.